xlators/mgmt/glusterd/src/glusterd-volume-set.c

```diff
@@ -2932,6 +2930,16 @@ struct volopt_map_entry glusterd_volopt_map[] = {
                     "brick multiplexing. Brick multiplexing ensures that "
                     "compatible brick instances can share one single "
                     "brick process."},
+    {.key = GLUSTERD_BRICK_GRACEFUL_CLEANUP,
+     .voltype = "mgmt/glusterd",
+     .value = "disable",
+     .op_version = GD_OP_VERSION_9_0,
+     .validate_fn = validate_boolean,
+     .type = GLOBAL_DOC,
```

Comment:
I'm not sure if we should make this option public. I consider this experimental, so it should be enabled only by who really wants to test it. Once it's considered stable, the option will be set to "enabled" by default and shouldn't be touched by users.

Reply:
Ack, Ok with not being public.